config/locales/en.json

```
@@ -621,8 +621,8 @@
 	"errors.postal_required": "Postal is required",
 	"errors.phone_required": "Phone is required",
 	"conditions.confirm_delete": "Are you sure you want to delete this condition?",
-	"medications.confirm_delete": "Are you sure you want to delete this medications?",
-	"treatments.confirm_delete": "Are you sure you want to delete this treatments?",
+	"medications.confirm_delete": "Are you sure you want to delete this medication?",
+	"treatments.confirm_delete": "Are you sure you want to delete this treatments",
```

```suggestion
	"treatments.confirm_delete": "Are you sure you want to delete this treatment?",
```
I did it wrong, @jeffmaher! I removed the `?` instead of the `s` in my suggested change. For shame.

---

No prob! Thanks for catching!